### PR TITLE
[RFC] Ops suggested modifications

### PR DESCRIPTION
**Pull request by gmantele**
All these commits are modifications suggested by Ops IG during [ADQL-2.1's RFC](https://wiki.ivoa.net/twiki/bin/view/IVOA/ADQL21RFC#Operations%20Interest%20Group).

### CODE REVIEW

**Review by msdemlei — CHANGES_REQUESTED** [review]
I'm fine wit the changes, but for match to make sense, the VARCHAR-s in its signature would need to be VARCHAR(*)-s.

(where I believe TEXT as a generic string type would be a good addition to our informal type collection -- but that's for another day).

**Comment by gmantele** [comment]
> I'm fine wit the changes, but for match to make sense, the VARCHAR-s in its signature would need to be VARCHAR(*)-s.
> 
> (where I believe TEXT as a generic string type would be a good addition to our informal type collection -- but that's for another day).

Strictly speaking, this is true, although most of the DBMS accept `VARCHAR` with no parameter (in which case, it is equivalent to `VARCHAR(*)`. That said, I actually wonder whether we should not rather make this parameter optional for `CHAR`, `VARCHAR`, `BINARY` and `VARBINARY` in ADQL-2.1. Honestly, it is very convenient to write just `VARCHAR` instead of `VARCHAR(*)`.

`TEXT` is also convenient to write ; but that would be an addition to ADQL-2.1, which should not happen during RFC period unless we have strong reason for that. So, I agree let's do that in another ADQL version.

**Comment by msdemlei** [comment]
On Wed, Jul 26, 2023 at 03:18:30AM -0700, Grégory Mantelet wrote:
> rather make this parameter optional for `CHAR`, `VARCHAR`, `BINARY`
> and `VARBINARY` in ADQL-2.1. Honestly, it is very convenient to
> write just `VARCHAR` instead of `VARCHAR(*)`.

Interesting -- I wasn't aware of that (of course, I've been living
in the paradise of postgres and sqlite with their TEXT type).

Given that convention, from my side: feel free to merge.

**Review by mbtaylor — DISMISSED** [review]
Thank you for addressing my comments.  Looks good to me.

File: ADQL.tex [hunk 1]
```diff
@@ -2410,14 +2215,9 @@ \subsubsection{POLYGON}
...
-        t1.ra,
-        t1.dec + 5,
-        t1.ra  - 5,
-        t1.dec - 5,
-        t1.ra  - 5,
-        t1.dec + 5,
-        )
+    POLYGON(t1.ra    , t1.dec + 5,
+            t1.ra - 5, t1.dec - 5,
+            t1.ra - 5, t1.dec + 5,)
```
Comment (mbtaylor) [inline ADQL.tex]: Remove the filnal trailing comma here (error inherited from previous version).

**Review by mbtaylor — CHANGES_REQUESTED** [review]
Thanks, I think it improves readability (and may save paper).
All OK apart from one tiny bugfix required as noted.

File: ADQL.tex (hunk 1 above)
Comment (gmantele) [reply ADQL.tex]: Oops. I already fixed one like that, but I have not seen this one. Now fixed. Thanks.

**Review by mbtaylor — APPROVED** [review]

**Pull request merged**